vendor/github.com/google/cadvisor/info/v1/machine.go

```diff
@@ -263,6 +263,7 @@ func (m *MachineInfo) Clone() *MachineInfo {
 		NumSockets:       m.NumSockets,
 		CpuFrequency:     m.CpuFrequency,
 		MemoryCapacity:   m.MemoryCapacity,
+		SwapCapacity: 	  m.SwapCapacity,
```

Comment:
we can start maybe sending PR(s) to cadvisor to make sure it has all the bits and pieces to support swap?

Reply:
You commented too fast :D

Here's the fix: https://github.com/google/cadvisor/pull/3293 on upstream cAdvisor.
I'll also add it to the commit description.

Obviously, if we'd choose to go with this implementation, we would have delete this commit and wait for the fix to be merged.